generic/tests/cfg/timerclock.cfg

```diff
@@ -9,6 +9,9 @@
             type = rtc
         - tsc:
             type = tsc
+            test_name = tsc_before_rhel8
```

Comment:
Recommend define it like `tsc_test_name`, to avoid possible name conflict.

Reply:
Updated.